app/services/events/record.rb

```diff
@@ -369,6 +369,31 @@ def self.record_school_partnership_created_event!(author:, school_partnership:)
       ).record_event!
     end
 
+    def self.record_school_partnership_updated_event!(author:, school_partnership:, previous_delivery_partner:)
```

Comment:
can we include `modifications` here as well, as that is expected in "update" events?